org.eclipse.scanning.points/src/org/eclipse/scanning/points/AbstractScanPointIterator.java

```diff
@@ -28,6 +52,66 @@ public void setPyIterator(Iterator<IPosition> pyIterator) {
 		this.pyIterator = pyIterator;
 	}
 
+	protected Iterator<IPosition> createSpgCompoundGenerator(Iterator<?>[] iterators, Object[] regions,
+			String[] regionAxes, PyObject[] mutators) {
+		JythonObjectFactory<PyObject> excluderFactory = ScanPointGeneratorFactory.JExcluderFactory();
+		@SuppressWarnings("rawtypes")
+		JythonObjectFactory<SerializableIterator> cpgFactory = ScanPointGeneratorFactory.JCompoundGeneratorFactory();
+		List<PyObject> pyRegions = Arrays.asList(regions).stream().map(r -> makePyRoi(r)).collect(Collectors.toList());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1612)